datasets/hsip-lidar-us-cities.yaml

```diff
@@ -4,10 +4,6 @@ Documentation: https://github.com/hobuinc/hsip-lidar
 Contact: https://github.com/hobuinc/hsip-lidar
 ManagedBy: "[Hobu, Inc.](https://hobu.co)"
 UpdateFrequency: Periodically
-Collabs:

```

Comment:
@apprivet please review this dataset of HSIP urban areas (https://registry.opendata.aws/hsip-lidar-us-cities/) for inclusion into the ASDI catalog.